packages/phoenix-event-display/src/managers/three-manager/import-manager.ts

```diff
@@ -38,10 +40,12 @@ export class ImportManager {
    */
   constructor(
     clipPlanes: Plane[],
+    clipIntersection: boolean,
```

Comment:
Do we need this in the import manager? We only ever use it when we change the clipping as far as I understand.

Reply:
I think we do in order to apply properly the ongoing clipping to any new geometry we could load. See method `processGeometry`

Reply:
Thanks for the clarification.